handler/cantabular_metadata_export.go

```diff
@@ -152,14 +152,14 @@ func (h *CantabularMetadataExport) exportTXTFile(ctx context.Context, e *event.C
 		download.Private = url
 	}
 
-	download.URL = url
+	download.URL = h.generateDownloadURL(e, "txt")
 
 	return download, nil
 }
 
 func (h *CantabularMetadataExport) exportCSVW(ctx context.Context, e *event.CSVCreated, m dataset.Metadata, isPublished bool) (*dataset.Download, error) {
 	filename := h.generateCSVWFilename(e)
-	downloadURL := h.generateDownloadURL(e)
+	downloadURL := h.generateDownloadURL(e, "csv-metadata.json")
```

Comment:
Are you sure thats the correct extension for a csvw file ??????????????

Reply:
Yes, 100%

Reply:
https://github.com/ONSdigital/dp-download-service/blob/develop/service/service.go line 161 you can see the path. Yes, it seems completely daft. No, I have no idea why it is this.

Reply:
Yes, the download service has some 'funny' things. I think that's why they want to refurbish it.

Reply:
Hmmm. OK ... thats what it is then.